distribution/scripts/jenkins-essentials.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euo pipefail
+
+passwordFileLocation="$JENKINS_HOME/secrets/initialAdminPassword"
+passwordFileLocationDirectory="$( dirname "$passwordFileLocation" )"
+
+echo -n "Creating $passwordFileLocationDirectory... "
+mkdir -p "$passwordFileLocationDirectory"
+echo "Done."
+
+echo -n "Generating admin password... "
+echo $RANDOM | md5sum | cut -d ' ' -f 1 > "$passwordFileLocation"
+echo "Done. Password value stored in $passwordFileLocation file."
```

Comment:
How does the user actually get access to that file? Do they need to `docker-cp` or something?

Reply:
As it is, `docker exec cat …` seems slightly simpler, but basically yes, that is the idea.